build.gradle

```diff
@@ -300,7 +300,7 @@ dependencyManagement {
 
 dependencies {
     implementation group: 'com.github.hmcts', name: 'et-common', version:'1.0.15'
-    implementation group: 'com.github.hmcts', name: 'et-data-model', version:'1.8.0'
+    implementation group: 'com.github.hmcts', name: 'et-data-model', version:'1.8.2'
```

Comment:
@paul-peasegood latest version is 1.9.0 but the tag for et-data-model hasn't been created yet from master, may be create the tag 1.9.0 and update it here as well.

Reply:
I would actually keep this at 1.8.2. If this version of data model contains everything we need, theres no need to use a later version. Also helps with history tracing if we ever need to roll back in future (ie, this code only needed 1.8.2 version of data model)

Reply:
ok we would still need a new tag 1.9.0 in et-data-model to match the Gradle.build version to be consistent. @paul-peasegood

Reply:
Agreed, a new tag needs to be made (and will need to be for whatever ticket uses it) but this is outside of this PR